docs/content/concepts/sui-architecture/consensus.mdx

```diff
@@ -8,11 +8,34 @@ The basic purpose of consensus in blockchains is to agree on a consistent order
+Sui uses the Mysticeti protocol to optimize for both low latency and high throughput. The benefits of Mysticeti include:
+
+- Supports multiple validators proposing blocks in parallel, utilizing the full bandwidth of the network and providing censorship resistance. These are features of the DAG-based consensus protocols.
+- The first consensus protocol to require only three rounds of messages to commit blocks from the DAGs (first of its kind), same as practical Byzantine Fault Tolerance and matches the theoretical minimum.
 - The commit rule allows voting and certifying leaders on blocks in parallel, further reducing the median and tail latencies.
 - The commit rule also tolerates unavailable leaders without significantly increasing the commit latencies.
 
-For more details including correctness proofs, the [Mysticeti paper](https://arxiv.org/pdf/2310.14821) is the best source.
+## Transaction throughput
+
+Compared to other state-of-the-art consensus protocols, Mysticeti is theoretically capable of handling over twice as many transactions with half the observed latency as other protocols. In a controlled environment using 10 nodes, Mysticeti is capable of handling 300,000 transactions per second (TPS) before latency crosses the one-second marker. Increased to 50 nodes, test results show 400,000 TPS before surpassing a one-second latency. In the same tests, the other top performing consensus mechanisms do not reach 150,000 TPS and observed latency _begins_ at around two seconds.
+
+On average, testing shows Mysticeti can achieve consensus commitment in about **0.5 seconds** with a sustained throughput of **200,000 TPS**.
+
+![Throughput and latency graph](./images/thruput_latency.png)
+
+## Decision rule
+
+The novel decision rule at the heart of Mysticeti optimizes its operational efficiency. Traditional consensus engine decision rules require explicit block validation and certification. This process necessitates greater communication overhead as validators sign and broadcast votes to reach consensus. By contrast, Mysticeti provides implicit commitment, which reduces this inter-node communication, significantly lowering bandwidth usage.
+
+## Finality
+
+Finality is the guarantee that a transaction or block, after confirmation, is permanently added to the network and cannot be altered or reversed. In traditional blockchain consensus, confirming transactions can take time because they rely on other transactions to “reference” them repeatedly before they are considered final. This process can slow down if the network activity decreases or if there are many competing transactions. In contrast, Mysticeti simplifies this process by finalizing transactions immediately upon inclusion in the structure. Consequently, there's no waiting for additional confirmations or network activity, making Mysticeti faster and more reliable for confirming transactions, even in less active or challenging network conditions.
```

Comment:
My understanding is here that we are comparing Mysticeti to chains mostly with non BFT chains like Bitcoin etc base safety on confirmations (well I guess some BFT consensus systems also use similar concepts) . If that's the intention here then that looks good to me. If we want to make more direct comparison with other state of the art consensus engines then we might want to be more specific. That being said, depending on the intention to remain more high level that's fine.

Reply:
That was the original intention. The idea is that this topic remains high and anyone wanting more details can reference the whitepaper.